treeherder/webapp/api/performance_data.py

```diff
@@ -226,3 +242,45 @@ class AlertPagination(pagination.CursorPagination):
         page_size = 10
 
     pagination_class = AlertPagination
+
+    def create(self, request, *args, **kwargs):
+        data = request.data
+        if 'summary_id' not in data or 'signature_id' not in data:
+            return Response({"message": "Summary and signature ids necessary "
+                             "to create alert"}, status=400)
+
+        summary = PerformanceAlertSummary.objects.get(
+            id=data['summary_id'])
+        signature = PerformanceSignature.objects.get(
+            id=data['signature_id'])
+        prev_data = PerformanceDatum.objects.filter(
+            signature=signature,
+            result_set_id__lte=summary.prev_result_set_id).order_by(
+                'push_timestamp').values_list('value', flat=True)[:10]
```

Comment:
Actually, as discussed on irc, we should use the signature values for this, and the perfherder defaults if not. We should be using the same algorithm for generating the change % with manually created alerts as we would for automatically created ones.